UsersGitHub/UsersGitHub.Android/CurrentVersion.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Android.Content.PM;
+using UsersGitHub.Interfaces;
+
+[assembly: Xamarin.Forms.Dependency(typeof(UsersGitHub.Droid.CurrentVersion))]
+namespace UsersGitHub.Droid
+{
+    public class CurrentVersion : IAppVersion
```

Comment:
also please rename CurrentVersion to CurrentVersionService and IAppVersion to IAppVersionService

Reply:
same for iOS